### PR TITLE
Add System Python UDFs to yql agent

### PR DESCRIPTION
**Pull request by Krock21**
A branch with enabled CI (this PR + CI enable commit): https://github.com/ytsaurus/ytsaurus/tree/members/krock21/systempython-udf-1-test-workflow

Tested on internal clusters([CI run](https://github.com/ytsaurus/ytsaurus/actions/runs/10198298190)). It works

### CODE REVIEW

**Comment by Krock21** [comment]
CI run: https://github.com/ytsaurus/ytsaurus/actions/runs/10198191440

**Comment by Krock21** [comment]
Rebased to pick up the latest YDB commit
CI run: https://github.com/ytsaurus/ytsaurus/actions/runs/10198298190

**Comment by Krock21** [comment]
CI run: https://github.com/ytsaurus/ytsaurus/actions/runs/10198298190

Tested on internal cluster. Works with `execute_udf_locally_if_possible = %false` in yql agent gateway spec

**Comment by Krisha11** [comment]
Lets make it false in trunk to make it workable in result builds?

https://github.com/ytsaurus/ytsaurus/blob/main/yt/yql/agent/config.cpp#L270

**Comment by Krock21** [comment]
> Lets make `execute_udf_locally_if_possible` `false` in trunk to make it workable in result builds?
> 
> https://github.com/ytsaurus/ytsaurus/blob/main/yt/yql/agent/config.cpp#L270

https://github.com/ytsaurus/ytsaurus/pull/754

File: yt/docker/ytsaurus/Dockerfile [hunk 1]
```diff
@@ -116,6 +116,9 @@ RUN chmod 755 /setup_cluster_for_chyt.sh
 
 FROM base-ytsaurus-python-packages AS query-tracker-bloated
 
+# Libpython shared objects. TODO(aleksandr.gaev) use stubs instead.
```
Comment (Krisha11) [inline yt/docker/ytsaurus/Dockerfile:119]: What does this TODO mean?

Comment (Krock21) [reply yt/docker/ytsaurus/Dockerfile:119]: It means that we don't need working python and `libpython3.N.so` files in this image, we only need stubs that declare required symbols

I was creating stubs using this script(passing libpython3.N.so as a first argument):
```
#/usr/bin/env bash
SYMBOLS=$(nm -D --defined-only "$1" | awk '{print $3}')
NAME=$(basename "$1")
echo "#include <stdlib.h>" > ${NAME}_stub.c
for SYMBOL in $SYMBOLS
do
        echo "void $SYMBOL(){exit(42);};" >> ${NAME}_stub.c
done

cc -shared ${NAME}_stub.c -o ${NAME}
```
and it worked on an internal cluster

Comment (Krock21) [reply yt/docker/ytsaurus/Dockerfile:119]: We may implement something similar to `YQL-7971`, when stubs are created automatically and added to yql_agent
In my tests I added them to `/usr/local/lib` and it made it worse for some other tools that required libpython (gdb)

**Review by Krisha11 — APPROVED** [review]

**Pull request closed without merging**